google/cloud/bigtable/internal/rpc_policy_parameters.inc

```diff
@@ -0,0 +1,89 @@
+// AUTOGENERATED BY tools/generate_rpc_policy_parameters.py; DO NOT EDIT
+// MANUALLY. ALL MANUAL CHANGES WILL BE OVERWRITTEN.
+//
+// Copyright 2017 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Interface name: "google.bigtable.admin.v2.BigtableInstanceAdmin"
+// Define the defaults using a pre-processor macro, this allows
+// the application developers to change the defaults for their
+// application by compiling with different values.
+
+#ifndef BIGTABLEINSTANCEADMIN_DEFAULT_INITIAL_DELAY
```

Comment:
I think these macros should be prefixed by `GOOGLE_CLOUD_CPP_`, they get very long, but that seems safer.

Reply:
Added prefix.